src/main/java/frc/robot/subsystems/lights/Lights.java

```diff
@@ -1,7 +1,91 @@
 package frc.robot.subsystems.lights;
 import frc.robot.Constants;
+import com.ctre.phoenix.led.CANdle;
+import com.ctre.phoenix.led.CANdle.LEDStripType;
+import com.ctre.phoenix.led.CANdleConfiguration;
+import com.ctre.phoenix.led.RainbowAnimation;
+import com.ctre.phoenix.led.StrobeAnimation;
 import frc.robot.RobotMap;
+import java.util.TreeMap;
 
 public class Lights {
-    
-}
+  /**
+   * Tree map of light code enums and integers that represent the R, G, B values of each light code
+   * enum
+   */
+  private TreeMap<LightCode, Integer[]> lightOptionsMap;
+
+  private CANdle candle = new CANdle(RobotMap.CANDLE_CAN_ID);
+  private CANdleConfiguration config = new CANdleConfiguration();
+  private LightCode currentLightStatus = LightCode.OFF;
+
+  public enum LightCode {
+    READY_TO_INTAKE, //ORANGE
+    HAS_CORAL, //GREEN
+    SCORE_PREP, //BLUE
+    READY_TO_SCORE, //YELLOW
+    CLIMB_PREP, //BLACK
+    READY_TO_CLIMB, //PURPLE
```

Comment:
I would remove CLIMB_PREP and instead add colors for CLIMB_PREP_SHALLOW and CLIMB_PREP_DEEP. In my opinion, READY_TO_CLIMB can be the same for both